docker-compose.yml

```diff
@@ -23,21 +23,31 @@ services:
     command: /bin/bash -c "service redis-server start && source /etc/profile.d/* && ant code-check"
   test:
     image: talis/talis-php
+    network_mode: host
```

Comment:
Needed to access persona.talis.local.

Reply:
Alternatively, you could add
```
  test:
    # ...
    networks:
      - default
      - infra

# ...
networks:
  infra:
    external:
      name: talis_shared
```

Reply:
I think the `host` setting is OK here. This isn't a docker container that's ever going to come up as part of the new docker environment for primitives and apps etc. It's just a docker container to run the tests for a client library on.